.renovaterc.json5

```diff
@@ -2,7 +2,8 @@
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
     "github>UCL-ARC/.github//renovate/default-config.json",
-    "schedule:monthly",
```

Comment:
the UCL-ARC config already extends from `schedule:monthly`

Reply:
so what's interesting is that it isn't running on a schedule. I had a hunch it wasn't being ran, so went to the ui and made the mistake of selecting all thinking that we had set up the grouping. Not high up on our list of issues though

Reply:
oh strange, maybe I'll take a quick look next month if it doesn't run automatically